pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -397,6 +397,7 @@ profiles:
 }
 
 func TestCodecsEncodePluginConfig(t *testing.T) {
+	parallelism := int32(8)
```

Comment:
```go
func i32ptr(n int32) *int32 {
  return &n
}
```

is a more common pattern to create a pointer to a int literal

Reply:
Actually, line 411 should just use `pointer.Int32ptr`

Reply:
Please do this